src/Sylius/Behat/Client/ApiPlatformSecurityClient.php

```diff
@@ -19,19 +19,27 @@
 
 final class ApiPlatformSecurityClient implements ApiSecurityClientInterface
 {
+    /** @var array<string, string|object> */
     private array $request = [];
 
     public function __construct(
-        private AbstractBrowser $client,
-        private SharedStorageInterface $sharedStorage,
-        private string $apiUrlPrefix,
-        private string $section,
+        private readonly AbstractBrowser $client,
+        private readonly SharedStorageInterface $sharedStorage,
+        private readonly string $apiUrlPrefix,
+        private readonly string $section,
+        private readonly string $resource,
```

Comment:
I am uncertain about the `resource` variable. My suggestion is to leave as it was and pass the`admin/administrator` or `shop/customer` value to the `section` variable. WDYT?